full_install.sh

```diff
@@ -0,0 +1,26 @@
+deactivate
+rm -rf ./venv
+python3 -m venv ./venv
+source ./venv/bin/activate
+pip install -U pip
+pip install mxnet-cu102
+pip install numpy
+pip install gluonnlp==0.8.3
+pip install torch
```

Comment:
Why install torch, torchvision, gluonnlp, numpy, wheel? Why uninstall typing?

Reply:
Typing is mac-only issue: it conflicts with python local installation. Other dependencies are needed for tests.